src/commands/FieldCommandConfirmMessage.js

```diff
@@ -0,0 +1,17 @@
+import { TextAreaFieldFF, ReactFinalForm } from '@dhis2/ui'
+import React from 'react'
+
+import { FIELD_COMMAND_CONFIRM_MESSAGE_NAME } from './fieldNames'
+import { dataTest } from '../dataTest'
+import i18n from '../locales'
+
+const { Field } = ReactFinalForm
+
+export const FieldCommandConfirmMessage = () => (
+    <Field
+        dataTest={dataTest('forms-fieldcommandconfirmmessage')}
```

Comment:
completely irrelevant now (and I assume that you just copied it from where I did it wrong as well):
the data test prop should be `commands-fieldcommandconfirmmessage`

Reply:
Cool, updated it 👍